R/addin.R

```diff
@@ -9,7 +9,7 @@
 #' @inheritParams use_template
 #'
 #' @export
-use_addin <- function(addin = "new_addin", open = interactive()) {
+use_addin <- function(addin = "new_addin", open = rlang::is_interactive()) {
```

Comment:
I namespace this when it appears as a user-facing default.

Otherwise I don't because we import the rlang namespace.